### PR TITLE
Folder contents

### PR DESCRIPTION
**Pull request by erasche**

### CODE REVIEW

File: bioblend/galaxy/folders/__init__.py [hunk 1]
```diff
@@ -31,18 +31,22 @@ def create_folder(self, parent_folder_id, name, description=None):
...
-    def show_folder(self, folder_id):
+    def show_folder(self, folder_id, contents=False):
         """
         Display information about a folder.
 
         :type folder_id: str
         :param folder_id: the folder's encoded id, prefixed by 'F'
 
+        :type contents: bool
+        :param contents: True if want to get contents of the folder (rather
```
Comment (bgruening) [inline bioblend/galaxy/folders/__init__.py]: `True to get the contents of the folder`?

File: bioblend/_tests/TestGalaxyFolders.py [hunk 2]
```diff
@@ -26,6 +26,12 @@ def test_show_folder(self):
         f2 = self.gi.folders.show_folder(self.folder['id'])
         self.assertEqual(f2['id'], self.folder['id'])
 
+    def test_show_folder_contents(self):
```
Comment (bgruening) [inline bioblend/_tests/TestGalaxyFolders.py]: @test_util.skip_unless_galaxy("release_16.01") ?

File: bioblend/galaxy/folders/__init__.py (hunk 1 above)
Comment (erasche) [inline bioblend/galaxy/folders/__init__.py]: copied and pasted contents from another description, will update.

File: bioblend/_tests/TestGalaxyFolders.py (hunk 2 above)
Comment (erasche) [inline bioblend/_tests/TestGalaxyFolders.py]: wow, this is so new. I was looking at the code and the surrounding bits were very old

File: bioblend/_tests/TestGalaxyFolders.py [hunk 3]
```diff
@@ -25,6 +25,13 @@ def test_create_folder(self):
     def test_show_folder(self):
         f2 = self.gi.folders.show_folder(self.folder['id'])
         self.assertEqual(f2['id'], self.folder['id'])
+
```
Comment (nsoranzo) [inline bioblend/_tests/TestGalaxyFolders.py]: ./bioblend/_tests/TestGalaxyFolders.py:28:1: W293 blank line contains whitespace

File: bioblend/_tests/TestGalaxyFolders.py (hunk 2 above)
Comment (nsoranzo) [inline bioblend/_tests/TestGalaxyFolders.py]: Actually only some metadata key/value pairs were added in 16.01, I've changed the test and doc accordingly.

Comment (erasche) [inline bioblend/_tests/TestGalaxyFolders.py]: Ah ok, brilliant. thank you.

**Pull request merged**